src/drive/web/modules/public/LightFileViewer.spec.jsx

```diff
@@ -43,6 +45,8 @@ describe('LightFileViewer', () => {
       ) // This is the sharing banner
       expect(queryByTestId('public-toolbar')).toBeTruthy()
       expect(queryByTestId('viewer-toolbar')).toBeFalsy()
+
+      consoleSpy.mockRestore()
```

Comment:
nit: on peut peut-être plus simplement faire un 

```
 beforeEach(() => {
    jest.clearAllMocks()
  })
```

Reply:
I totally agree, each test files should clean mock by default https://github.com/cozy/cozy-drive/pull/2508